cypress/support/fragments/orders/orders.js

```diff
@@ -34,9 +34,9 @@ const searchButton = Button('Search');
 const newButton = Button('New');
 const saveAndClose = Button('Save & close');
 const orderDetailsAccordionId = 'purchaseOrder';
-const createdByAdmin = 'ADMINISTRATOR, DIKU ';
+const createdByAdmin = 'folio-aqa  ';
```

Comment:
Please use more universal approach. Take the Last name[, First name] from the profileDropdown